controllers/grid/queries/QueriesGridHandler.inc.php

```diff
@@ -762,7 +761,7 @@ public function fetchTemplateBody(array $args, PKPRequest $request): JSONMessage
         if ($template) {
             $user = $request->getUser();
             $template->assignParams([
-                'editorialContactSignature' => $user->getContactSignature(),
+                'editorialContactSignature' => $user->getSignature() ?? '',
```

Comment:
I think you need to add `Locale::getLocale()` here.

Reply:
THanks @NateWr !

I have changed the method parameter by adding the Locale::getLocale()

Reply:
@NateWr 
sorry, thanks a lot  for finding that. My editor missed that and was pointing to another  Locale.
I have added the changes and pushed it.